### PR TITLE
improve localization

### PR DESCRIPTION
**Pull request by TheBestTvarynka**

### CODE REVIEW

File: frontend/src/components/SwitchRoleModal/index.tsx [hunk 1]
```diff
@@ -69,7 +69,9 @@ const SwitchRoleModal: FC<ISwitchRoleModalProps> = (
                         <Header>{t("Select new role")}</Header>
                         <Select compact
                                 className={styles.dropdown}
-                                options={companyRoles.map(r => ({key: r.id, value: r.id, text: r.name}))}
+                                options={companyRoles.map(r => ({
+                                  key: r.id, value: r.id, text: t(r.name.toLowerCase())
```
Comment (TheBestTvarynka) [inline frontend/src/components/SwitchRoleModal/index.tsx:73]: We also translate the role name.

File: frontend/src/localization/languages/ukrainian.json [hunk 2]
```diff
@@ -127,6 +128,7 @@
     "Fire": "Звільнити",
     "Switch role": "Змітини роль",
     "company_owner": "власник компанії",
+    "company owner": "власник компанії",
     "hr": "рекрутер",
     "employee": "працівник",
```
Comment (grytsiuko) [inline frontend/src/localization/languages/ukrainian.json:136]: Why to insert "company owner" key twice?

Comment (TheBestTvarynka) [reply frontend/src/localization/languages/ukrainian.json:136]: First one when we render role name on page. Second one when we rednder role name on modal window.

Comment (grytsiuko) [reply frontend/src/localization/languages/ukrainian.json:136]: Okay, I didn't notice underline

**Review by grytsiuko — APPROVED** [review]

**Comment by lenazelenaya** [comment]
image

**Pull request merged**